library/Rules/MinimumAge.php

```diff
@@ -1,36 +0,0 @@
-<?php
-namespace Respect\Validation\Rules;
-
-use DateTime;
-
-class MinimumAge extends AbstractRule
```

Comment:
Do the same was done in [Vowels](https://github.com/Respect/Validation/blob/master/library/Rules/Vowels.php#L9-L10) rule, for example, them we can remove it in the next major release.